__tests__/app.test.js

```diff
@@ -47,3 +49,44 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with an object with all relevant properties", () => {
+    return request(app)
+      .get(`/api/articles/1`)
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((column) => {
+          expect(typeof column.author).toBe("string");
+          expect(typeof column.title).toBe("string");
+          expect(typeof column.article_id).toBe("number");
+          expect(typeof column.body).toBe("string");
+          expect(typeof column.topic).toBe("string");
+          expect(typeof column.created_at).toBe("string");
+          expect(typeof column.votes).toBe("number");
+          expect(typeof column.article_img_url).toBe("string");
+        });
+      });
+  });
+});
+
+describe("Err /api/articles/article_id", () => {
```

Comment:
400+ status codes usually still go to the same describe block as they belong to the same endpoint, but if this makes your code more readable for you, it's fine to go with it.